### PR TITLE
Flake8 fixes pt1

### PR DESCRIPTION
**Pull request by cadenmyers13**

### CODE REVIEW

File: src/diffpy/pdfgui/gui/windowperspective.py [hunk 1]
```diff
@@ -15,4 +15,40 @@
...
...00;maxw=-1;maxh=-1;floatx=-1;floaty=-1;floatw=-1;floath=-1|name=rseries;caption=;state=770;dir=5;layer=0;row=0;pos=0;prop=100000;bestw=400;besth=380;minw=190;minh=200;maxw=-1;maxh=-1;floatx=-1;floaty=-1;floatw=-1;floath=-1|name=tseries;caption=;state=770;dir=5;layer=0;row=0;pos=0;prop=100000;bestw=400;besth=380;minw=190;minh=200;maxw=-1;maxh=-1;floatx=-1;floaty=-1;floatw=-1;floath=-1|name=outputPanel;caption=PDFfit2 Output;state=10487804;dir=3;layer=0;row=0;pos=0;prop=100000;bestw=400;besth=40;minw=200;minh=40;maxw=-1;maxh=-1;floatx=-1;floaty=-1;floatw=-1;floath=-1|name=treeCtrlMain;caption=Fit Tree;state=10487804;dir=4;layer=0;row=0;pos=0;prop=100000;bestw=200;besth=100;minw=200;minh=40;maxw=-1;maxh=-1;floatx=471;floaty=730;floatw=208;floath=124|name=plotPanel;caption=Plot Control;state=10487804;dir=4;layer=0;row=0;pos=1;prop=100000;bestw=200;besth=250;minw=200;minh=150;maxw=-1;maxh=-1;floatx=337;floaty=393;floatw=208;floath=274|name=journalPanel;caption=Project Journal;state=10487807;dir=4;layer=0;row=0;pos=0;prop=100000;bestw=450;besth=450;minw=200;minh=200;maxw=-1;maxh=-1;floatx=393;floaty=113;floatw=450;floath=450|dock_size(3,0,0)=59|dock_size(4,0,0)=202|dock_size(5,0,0)=362|"
```
Comment (sbillinge) [inline src/diffpy/pdfgui/gui/windowperspective.py:18]: This one we probably don't want to split up.  You can restore the old file by doing `git checkout <filename> in the right branch. Be very careful with this.  It is the only time that got will let you overwrite a local file and potentially lose work.  But we want to do that here. 

To get it to pass ci add `#noqa ignore E501` to the end of that line of code, but check the syntax on Google, I don't remember exactly.

**Review by sbillinge — COMMENTED** [review]
Please see comment.  O may have more but about to jump on a train

**Review by sbillinge — COMMENTED** [review]
I hate to say this, but I think these are all wrong.  We don't want to update copyright statements to either MSU or BNL, only adding/updating statements to Columbia.

The basic idea is that the copyright is owned by the entity that made the changes. Right now, since we are Columbia, Columbia owns copyright to changes we are making to the code now, but BNL/MSU don't have copyright for edits we are doing now (but still have copyright for the work done before), if you see what I mean.

**Comment by sbillinge** [comment]
reopening.  Sorry, we just need to revert the copyright work on this PR.  I think the rest is ok.

Let's put all the copyright work on separate PRs.

**Comment by cadenmyers13** [comment]
> I hate to say this, but I think these are all wrong.  We don't want to update copyright statements to either MSU or BNL, only adding/updating statements to Columbia.
> 
> 
> 
> The basic idea is that the copyright is owned by the entity that made the changes. Right now, since we are Columbia, Columbia owns copyright to changes we are making to the code now, but BNL/MSU don't have copyright for edits we are doing now (but still have copyright for the work done before), if you see what I mean.

Ahh. I was wrestling with this earlier trying to revert my commits back to fix this. I thought I got it fixed in the PR but looks like I didn't😭. I will get this fixed. It's not too much work since I can search globally (command+f) on VS code and replace.

**Comment by sbillinge** [comment]
this branch has conflicts too that needs resolving.  We are approaching a point where it may be better to open  a clean branch and redo the edits...... Especially since this will also include hte new work to get CI running.

**Pull request closed without merging**